scripts/build-images.sh

```diff
@@ -20,13 +20,13 @@ COMMIT_SHA=${LND_VERSION}
 LND_BTC_NODE=bitcoind
 LND_LTC_NODE=litecoind
 
-# NOTE: The names specified with `-t` directly map to the service names in
-# the applicable services docker-compose file
-docker build -t sparkswap_lnd_btc ./docker/lnd --build-arg NODE=$LND_BTC_NODE --build-arg NETWORK=btc --build-arg COMMIT_SHA=$COMMIT_SHA
-docker build -t sparkswap_lnd_ltc ./docker/lnd --build-arg NODE=$LND_LTC_NODE --build-arg NETWORK=ltc --build-arg COMMIT_SHA=$COMMIT_SHA
+# NOTE: The names specified with `-t` directly map to the our service names in
+# on sparkswap dockerhub
+docker build -t sparkswap/lnd_btc:latest ./docker/lnd --build-arg NODE=$LND_BTC_NODE --build-arg NETWORK=btc --build-arg COMMIT_SHA=$COMMIT_SHA
+docker build -t sparkswap/lnd_ltc:latest ./docker/lnd --build-arg NODE=$LND_LTC_NODE --build-arg NETWORK=ltc --build-arg COMMIT_SHA=$COMMIT_SHA
```

Comment:
Should we tag this with `latest` by default? Or tag it with the current version from package.json? Or the commit hash?

Is this how we publish images as well, or is this just building, with publishing requiring a separate `tag` step?

Reply:
@treygriffith we're going to have to rethink our docker setup but here is why we tag latest:

- by default, if we omit the tag it goes to latest
- our docker-compose files use latest (which is easier *for us* than having them as a specific version)

By having all images tagged to latest, `latest` can then mean `latest sparkswap version` OR `the version I build on my machine`

With this said, the more and more we lock these version (including locking them in broker/relayer docker-compose), the harder it will be to dev so we'll have to take that into consideration)

TBH we're probably better off tagging the version from package.json or something.

RE: your last comment, publishing does require a separate tag step (so we can technically have a different name locally than on dockerhub)

Reply:
@treygriffith TBH the thing im running into is that even though the code for lnd-engine may change, that doesnt mean the image has changed.

It feels weird to continually tag an image `v0.5.0`, `v0.5.1` when the images have had no changes